src/App.css

```diff
@@ -59,3 +60,28 @@ ul {
 .checkbox {
   margin: 10px;
 }
+.background {
+  position: absolute;
+  background-color: rgba(41, 43, 45, 0.75);
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: 10;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.modal {
+  background-color: white;
+  border-radius: 10px;
+  padding: 20px;
+  position: relative;
+}
+
+.back-button {
+  position: absolute;
+  left: 15px;
+  top: 15px;
+  padding: 10px;
```

Comment:
I'm wondering if it would make sense to switch the `px` to `rem` units to scale for users who change their font-sizes

Reply:
My take away from that accessibility presentation was that we want font-related sizing to scale, but layout-related sizing to remain constant. I don't think we'd want the positioning or padding of the button to scale when increasing the font size so I chose px over rem.